tests/Fixtures/Entity/RichPost.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+namespace Zenstruck\Foundry\Tests\Fixtures\Entity;
+
+use Doctrine\Common\Collections\ArrayCollection;
+use Doctrine\Common\Collections\Collection;
+use Doctrine\ORM\Mapping as ORM;
+
+/**
+ * @ORM\Entity
+ */
+class RichPost
+{
+    /**
+     * @ORM\Id
+     * @ORM\GeneratedValue
+     * @ORM\Column(type="integer")
+     */
+    private $id;
+
+    /**
+     * @ORM\Column(type="string", length=255)
+     */
+    private $title;
+
+    /**
+     * @ORM\OneToMany(targetEntity=RichComment::class, mappedBy="post")
+     */
+    private Collection $comments;
```

Comment:
This typehint causes problems when running tests on < PHP 7.4...

Reply:
I think we can skip the related test for lower versions

Reply:
It's not that easy as the `RichPost` will be loaded on every version.

Reply:
I got another idea :joy: 
https://www.php.net/supported-versions.php

Reply:
I got another idea :imp: 
![Capture d’écran du 2022-10-13 17-40-32](https://user-images.githubusercontent.com/10139766/195642558-72146475-e689-4ccd-a1dc-956c5ce6d52b.png)

Reply:
I know. I think we should bump the min PHP version to 8.0 in `1.x` and leave a `1.?.x` branch for bug fixes.

Reply:
I'm a lot in favor for this move :+1: